io-engine/src/subsys/nvmf/admin_cmd.rs

```diff
@@ -9,60 +9,65 @@ use std::{
 
 use crate::{
     bdev::nexus,
-    core::{logical_volume::LogicalVolume, Bdev, Reactors},
-    lvs::{lvs_lvol::LvsLvol, Lvol},
+    core::{
+        logical_volume::LogicalVolume,
+        snapshot::SnapshotOps,
+        Bdev,
+        Reactors,
+        SnapshotParams,
+    },
+    lvs::Lvol,
 };
-
 use spdk_rs::{
     libspdk::{
         nvme_cmd_cdw10_get,
         nvme_cmd_cdw10_get_val,
         nvme_cmd_cdw11_get,
         nvme_cmd_cdw11_get_val,
-        nvme_status_get,
         spdk_bdev,
         spdk_bdev_desc,
         spdk_bdev_io,
         spdk_io_channel,
         spdk_nvme_cmd,
         spdk_nvme_cpl,
-        spdk_nvme_status,
         spdk_nvmf_bdev_ctrlr_nvme_passthru_admin,
         spdk_nvmf_request,
         spdk_nvmf_request_get_bdev,
         spdk_nvmf_request_get_cmd,
-        spdk_nvmf_request_get_response,
         spdk_nvmf_request_get_subsystem,
         spdk_nvmf_set_custom_admin_cmd_hdlr,
         spdk_nvmf_subsystem_get_max_nsid,
     },
     nvme_admin_opc,
 };
 
+#[warn(unused_variables)]
 #[derive(Clone)]
 pub struct NvmeCpl(pub(crate) NonNull<spdk_nvme_cpl>);
-
-impl NvmeCpl {
-    /// Returns the NVMe status
-    pub(crate) fn status(&mut self) -> &mut spdk_nvme_status {
-        unsafe { &mut *nvme_status_get(self.0.as_mut()) }
-    }
-}
+/// TO-DO: Currently Snapshot Creation in Remote for Lvol is not supported,
```

Comment:
Can't we just remove this if it's not used?

Reply:
IMHO, we can keep it as potential code source for now, we can wipe out all unused code later once refactored.